docs/4.tutorials-and-examples/6.custom-webchat.mdx

```diff
@@ -115,24 +167,109 @@ export const MyHeader = props => {
 This is an example of a custom Header for our webchat. We can put all the fancy css styles we want.
 This is a static object, so it's not applied in any [RCML](/main-concepts/rich-conversational-markup-language) component, so we don't have to put `{props.children}` inside any container.
 
-The last thing we need to do, is to import all this custom components into our bot.
+The last thing we need to do, is to import all this custom components into our bot's webchat index.
 
-**src/app.js**
+**src/webchat/index.js**
 
 ```javascript
-import { App } from '@botonic/react'
+import { MyCustomReply } from './myCustomReply'
+import { MyCustomButton } from './myCustomButton'
+import { MyCustomHeader } from './myCustomHeader'
+import launcherIcon from '../assets/launcher-logo.png'
+import { MyCustomIntroImage } from './myCustomIntroImage'
+import { MyBotLogoChat } from './myBotLogoChat'
+import { MyCustomHeaderImage } from './myCustomHeaderImage'
+import { TriggerCustom } from './triggerCustom'
+
+export const webchat = {
+  theme: {
+    style: {
+      position: 'fixed',
+      right: 20,
+      bottom: 20,
+      width: 400,
+      height: 500,
+      margin: 'auto',
+      backgroundColor: 'white',
+      borderRadius: '25px',
+      boxShadow: '0 0 50px rgba(0,0,255,.30)',
+      backgroundImage:
+        'linear-gradient(to top, #ffffff,#ffffff 11%,#9a9ae3 40%,#0000ff 85%,#0000ff 85%)'
+    },
+    brandColor: '#0000ff',
+    textPlaceholder: 'Type something...',
+    botLogoChat: MyBotLogoChat,
+    headerTitle: 'My customized webchat',
+    headerSubtitle: 'R2D2',
+    headerImage: MyCustomHeaderImage,
+    introImage: MyCustomIntroImage,
+    customUserMessages: {
+      color: 'white',
+      borderRadius: '20px'
+    },
+    customBotMessages: {
+      color: 'blue',
+      borderRadius: '10px'
+    },
+    customReply: MyCustomReply,
+    alignReplies: 'right',
+    wrapReplies: 'wrap',
+    customButton: MyCustomButton,
+    brandIconUrl: launcherIcon,
+    customTriggerButton: TriggerCustom,
+    customHeader: MyCustomHeader
+  }
+}
+```
+
+All this changes can be tested using the `botonic serve` command *(`npm run start` on Windows)*. 
+Try to play with the properties in the template in order to see how they affect to the differents components in the webchat. 
```

Comment:
"Try to play" =>"try playing". In this case maybe we can use "Experiment with"

---------
try

I’m trying to learn Japanese but it’s very difficult.
Try + to + infinitive means that something is difficult and you make an attempt to do it.

Have you tried using butter instead of oil?
Try + ‘ing’ means make an experiment. It’s not difficult – it might work, it might not.

Reply:
"affect to" => "affect"

Reply:
Experiment with sounds more challenging